tests/e2e/scale/test_scale_3_OCS_worker_nodes_and_1500_PVCs.py

```diff
@@ -111,6 +111,70 @@ def test_pv_scale_out_create_pvcs_and_respin_ceph_pods(
         )
 
 
+@scale
```

Comment:
Please add bz marker

Reply:
oh!! I can add it, it's quick fix, but not re-running the test validation